specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/examples/CreateAccountWithEncryption.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "api-version": "2017-04-18",
+    "subscriptionId": "f9b96b36-1f5e-4021-8959-51527e26e6d3",
+    "resourceGroupName": "felixwa-01",
+    "accountName": "testCreate1",
+    "account": {
+      "location": "West US",
+      "kind": "Emotion",
```

Comment:
Emotion Kind has been decommissioned.

Reply:
this is swagger example, more for demonstrating/swagger validation purpose. Should be okay to use Emotion as sample kind.

Reply:
better to use valid kind. example files would be used to generate test/help or test the server. This should be correct.

Reply:
not only for swagger validation purpose.